yoda/src/main/proto/yoda.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
```

Comment:
this shouldn't exist (goto pom file for explanation) nor should it get checked into source control (never have 2 copies of the same data/configuration/code.. the source of truth for our proto definitions exist at <Jukebox repository root>/protos. No mirror of that information should be checked into source control.